### PR TITLE
Prima F Modified Graph

### PR DESCRIPTION
**Pull request by pancaprima**

### CODE REVIEW

File: locust/static/locust.js [hunk 1]
```diff
@@ -189,3 +216,26 @@ function updateExceptions() {
     });
 }
 updateExceptions();
+
+function createEndpointCharts(chartKey, callback) {
+  if(!endpointResponseTimeCharts[chartKey]) {
+    title = "Average Response Times"
+    endpointResponseTimeCharts[chartKey] = new LocustLineChart($(".charts-container"), title, chartKey, ["Average Response Time"], "ms", "33%");
```
Comment (rizalfr) [inline locust/static/locust.js]: why not 33.3 % ?

File: locust/static/locust.js [hunk 2]
```diff
@@ -189,3 +216,26 @@ function updateExceptions() {
...
+    endpointResponseTimeCharts[chartKey].resize()
+  }
+  if(!endpointRpsCharts[chartKey]) {
+    title = "Requests per Second"
+    endpointRpsCharts[chartKey] = new LocustLineChart($(".charts-container"), title, chartKey, ["RPS"], "request", "33%");
```
Comment (rizalfr) [inline locust/static/locust.js]: why not 33.3 % ?

File: locust/static/locust.js [hunk 3]
```diff
@@ -189,3 +216,26 @@ function updateExceptions() {
...
+    endpointRpsCharts[chartKey].resize()
+  }
+  if(!endpointFailureCharts[chartKey]) {
+    title = "Failures"
+    endpointFailureCharts[chartKey] = new LocustLineChart($(".charts-container"), title, chartKey, ["Failure"], "failure", "33%");
```
Comment (rizalfr) [inline locust/static/locust.js]: why not 33.3 % ?

File: locust/static/locust.js [hunk 4]
```diff
@@ -189,3 +216,26 @@ function updateExceptions() {
     });
 }
 updateExceptions();
+
+function createEndpointCharts(chartKey, callback) {
+  if(!endpointResponseTimeCharts[chartKey]) {
+    title = "Average Response Times"
```
Comment (rizalfr) [inline locust/static/locust.js]: I think this not the average, just response times

Comment (pancaprima) [reply locust/static/locust.js]: It's average, the one that using total is response time

File: locust/static/locust.js (hunk 3 above)
Comment (pancaprima) [reply locust/static/locust.js]: updated to 33.3%

File: locust/static/locust.js (hunk 2 above)
Comment (pancaprima) [reply locust/static/locust.js]: updated to 33.3%

File: locust/static/locust.js (hunk 1 above)
Comment (pancaprima) [reply locust/static/locust.js]: updated to 33.3%

**Pull request merged**